app/views/_add_to_playlist.html.erb

```diff
@@ -22,13 +28,14 @@ Unless required by applicable law or agreed to in writing, software distributed
   <% @add_playlist_item_playlists = Playlist.where(user: current_user) %>
   <span style="display:none" id='add_playlist_item_playlists_not_empty'><%= @add_playlist_item_playlists.empty? %></span>
   <% unless @add_playlist_item_playlists.empty? %>
+    <% @add_playlist_item_playlists = @add_playlist_item_playlists.sort_by(&:title) %>
```

Comment:
If no playlists were returned and I used sortby on that, would it break though? I guess it's just sorting an empty array so prob not.